x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -144,7 +144,9 @@ export default function ({
 
       afterEach(async () => {
         retry.waitFor('close share modal', async () => {
-          await PageObjects.share.closeShareModal(); // close modal
+          if (await testSubjects.exists('shareContextModal')) {
+            await PageObjects.share.closeShareModal(); // close modal
+          }
```

Comment:
We don't need this conditional, especially that the `closeShareModal` method on the share page object already performs this check. See https://github.com/elastic/kibana/commit/a3d42c407ee34e9cc0bceec0a8eb111bab5b7499#diff-59204f4a83a9ee3aaddd2e25fea3533dc768443e0639dc513e9e99a5097f7f76R20-R21